cli/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ _Released 11/7/2023 (PENDING)_
 **Bugfixes:**
 
 - Fixed an issue determining visibility when an element is hidden by an ancestor with a shared edge. Fixes [#27514](https://github.com/cypress-io/cypress/issues/27514).
+- Fixed an issue where `nextTestHasTestIsolationOn` was incorrectly being set when there isn't a next test. Addressed in [#28213](https://github.com/cypress-io/cypress/pull/28213)
```

Comment:
`test:before:after:run:async` and `nextTestHasTestIsolationOn` aren't publicly documented, so I'm not sure this is meaningful for an end user. Is there an effect this has or a bug this fixes that affects users?

Reply:
Good point. It _may_ help some protocol users where CDP events were still being processed at the end of the spec.

Reply:
Maybe I should update the text to be something like:
"Stopped processing CDP events at the end of a spec when Test Isolation is off and Test Replay is enabled."

Reply:
That sounds good

Reply:
updated [08acec7](https://github.com/cypress-io/cypress/pull/28213/commits/08acec70ce3450238a93d4ed14642b794506e54e)